include/lsst/afw/table/Simple.h

```diff
@@ -205,7 +206,7 @@ inline void SimpleRecord::setId(RecordId id) { set(SimpleTable::getIdKey(), id);
 
 inline IcrsCoord SimpleRecord::getCoord() const { return get(SimpleTable::getCoordKey()); }
 inline void SimpleRecord::setCoord(IcrsCoord const & coord) { set(SimpleTable::getCoordKey(), coord); }
-inline void SimpleRecord::setCoord(Coord const & coord) { set(SimpleTable::getCoordKey(), coord); }
+inline void SimpleRecord::setCoord(Coord const & coord) { SimpleTable::getCoordKey().set(*this, coord); }
 
```

Comment:
Does this convert coord to an IcrsCoord? I don't see it in the code. If it's not happening, it should (though I realize it's a pre-existing issue). If it is happening automagically then please document it.